### ISSUE TITLE
Add `configure` command

### ISSUE THREAD

**Issue by ilazakis** [Kenza-AI/sagify#59]
**tl;dr** Proposing the replacement of the `init` command with a `configure` command ala *aws configure* to allow for certain config values to change after the creation of a project.

**Issue**
The `init` command hardcodes most (all?) of its arguments throughout the project for the *cookiecutter* step to pick up later and replace in various places; for example in the [build script](https://github.com/Kenza-AI/sagify/blob/8923f1aab9e796e8b139d46c223ca37db44f601d/sagify/template/%7B%7B%20cookiecutter.module_slug%20%7D%7D/build.sh#L9).

This is causing issues when someone wants to change any of the initial arguments such as the project / image name, _python_ version etc

**Suggestion**
Addition of a `configure` command that can be called whenever someone wishes to change any of the original values. The command will effectively do exactly what `init` does but instead of hardcoding the values around the project via *cookiecutter* it will place them all in a local config file that can be updated every time the configure method is called again.

The other commands can consult that file to load their config each time they run.

**Comment by ilazakis** [comment]
Addressed by #72
